cmd/storagenode-updater/main.go

```diff
@@ -136,62 +160,72 @@ func cmdRun(cmd *cobra.Command, args []string) (err error) {
 	return nil
 }
 
-func update(ctx context.Context, nodeID storj.NodeID) (err error) {
-	client := checker.New(runCfg.ClientConfig)
+func cmdRecover(cmd *cobra.Command, args []string) error {
+	log.Println("storagenode updater recovering!")
+
+	badExec := strings.Replace(os.Args[0], ".backup", "", 1)
```

Comment:
please assign `os.Args[0]` to a variable with good name. here and in `renameUpdater`